bin/compile

```diff
@@ -8,11 +8,21 @@ echo "-----> Install ImageMagick"
 
 BUILD_DIR=$1
 CACHE_DIR=$2
+ENV_DIR=$3
 VENDOR_DIR="$BUILD_DIR/vendor"
 mkdir -p $VENDOR_DIR
 INSTALL_DIR="$VENDOR_DIR/imagemagick"
 mkdir -p $INSTALL_DIR
-IMAGE_MAGICK_VERSION="${IMAGE_MAGICK_VERSION:-6.9.5-10}"
+if [ -d $ENV_DIR ]; then
```

Comment:
Could we collapse this conditional and the one two lines down together into a single `-f` test? I think that would also make it so we only have to specify the default version once.

Reply:
like 

```bash
[ -f "${ENV_DIR}/IMAGE_MAGICK_VERSION" ] && export "IMAGE_MAGICK_VERSION=$(cat "${ENV_DIR}/IMAGE_MAGICK_VERSION")"
IMAGE_MAGICK_VERSION="${IMAGE_MAGICK_VERSION:-6.9.5-10}"
```